### DISCUSSION TITLE
Change default action when Newpipe is opened via hyperlink

### DISCUSSION THREAD

**Discussion by zwsren435** [TeamNewPipe/NewPipe#10853] [Questions]
When a youtube.com link is pressed, Newpipe shows a list of options. I accidentally checked a "do this option by default" checkbox on one of the options, and now Newpipe is automatically doing that option instead of showing the list. 

There is a back button on the Newpipe popup window, but it acts as an X button and closes the window instead of going back.

How can I have newpipe show the list again?

**Comment by opusforlife2** [comment]
Settings > Video and audio > Preferred open action.
